scripts/startup.sh

```diff
@@ -226,29 +228,252 @@ case ${options[$?]} in
 esac
 }
 
+manualpart () {
+  clear
+  echo -ne "
+------------------------------------------------------------------------
+    WARNING: CFDISK IS A COMMAND LINE DISK PARTITION MANAGEMENT TOOL.
+    IT MAY BE HAZARDOUS TO USE IT IF YOU ARE NOT FAMILIER WITH IT.
+    This might cost you all your data stored in this device.
+------------------------------------------------------------------------
+
+"
+
+  lsblk
+
+  echo -e "\nSelect the disk which you want to partition: "
+  options=($(lsblk -n --output TYPE,KNAME,SIZE | awk '$1=="disk"{print "/dev/"$2"|"$3}'))
+
+  select_option $? 1 "${options[@]}"
+  cfdisk ${options[$?]%|*}
+}
+
 # selection for disk type
 diskpart () {
+clear
 echo -ne "
 ------------------------------------------------------------------------
-    THIS WILL FORMAT AND DELETE ALL DATA ON THE DISK
+    WARNING: THIS MAY FORMAT AND DELETE ALL DATA ON THE DISK.
     Please make sure you know what you are doing because
-    after formating your disk there is no way to get data back
+    after formating your disk there is no way to get data back.
 ------------------------------------------------------------------------
 
 "
 
-PS3='
-Select the disk to install on: '
-options=($(lsblk -n --output TYPE,KNAME,SIZE | awk '$1=="disk"{print "/dev/"$2"|"$3}'))
+fdisk -l
 
+options=("Install in a specific Disk Partition" "Install in a whole Disk Drive" "Modify partitions using cfdisk")
 select_option $? 1 "${options[@]}"
-disk=${options[$?]%|*}
 
-echo -e "\n${disk%|*} selected \n"
-    set_option DISK ${disk%|*}
+case $? in
+
+0)
+  clear
+  echo -ne "
+------------------------------------------------------------------------
+    WARNING: ARCH LINUX WILL BE INSTALLED IN THE SELECTED PARTITION.
+    This will delete all the data present in it, so make sure that
+    you select the right partition that donot contain any useful files.
+------------------------------------------------------------------------
+
+
+"
+
+  fdisk -l
+
+  echo -e "\n"
+  echo "Select the Partition to Install on (This will be the root partition mounted on \"\\\"): "
+  options=($(lsblk -n --output TYPE,KNAME,SIZE,MOUNTPOINT | awk '$1=="part"{print "/dev/"$2"|"$3"__________"$4}'))
+  
+  select_option $? 1 "${options[@]}"
+  part=${options[$?]%|*}
```

Comment:
hay there, no a big thing but if you can change part and PART. Not a big thing but just bit confusing when reading code

Reply:
Well, the original had used disk and DISK as well, so I did the same.